src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class RightTriangle extends AbstractFigure {
+    private String color;
```

Comment:
The `color` field is redundant here because it is already defined in the `AbstractFigure` class. You should remove this field from the `RightTriangle` class to avoid confusion and potential errors.